frontend/app/src/components/FindUser.svelte

```diff
@@ -12,6 +12,7 @@
     import MatchingUser from "./MatchingUser.svelte";
     import Translatable from "./Translatable.svelte";
     import { trimLeadingAtSymbol } from "../utils/user";
+    import { Debouncer } from "../utils/debouncer";
```

Comment:
There is no functional change to this file, I just switched it to using the new `Debouncer` class rather than manually handling the debouncing logic.